autonomic-plugin-common/src/main/java/br/com/autonomiccs/autonomic/plugin/common/enums/AutonomiccsJarAbstraction.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
@alexandrelimassantana 
Could I suggest you a new name for this class? maybe JarsToBeDeployedInSystemVmsEnumeration ?
Ty